config/stamp/default/bvt.sh

```diff
@@ -188,3 +188,14 @@ NEWRELIC_INFRASTRUCTURE_LICENSE_KEY="ae20a4f63756f2369c483845deb6e7afdedf8e5e"
 COMMON_ENABLE_NEWRELIC_INFRASTRUCTURE=true
 #Insights
 EDXAPP_ANALYTICS_DASHBOARD_URL=""
+
+# Azure keyvault configs 
+EDXAPP_GET_SECRETS_FROM_AZURE_KEYVAULT=false
```

Comment:
Nit: Change these to double quotes to be consistent with the rest of the file.

Reply:
Done